src/modules.js

```diff
@@ -40,14 +47,20 @@ if (__analytics.googleTagManager?.isEnabled) {
   const googleAnalyticsModule = function () {
     /* eslint-disable */
     // @formatter:off
```

Comment:
@AlanLes you should use setup `@formatter:off` in your IDE because this parts of code shouldn't be formatted